.github/workflows/test.yml

```diff
@@ -51,11 +51,14 @@ jobs:
         cargo install --version 0.9.1 wasm-pack
     - name: Build
       run: cargo build --verbose $TARGET
+    - name: Disable benchmarking dependencies
+      if: matrix.mode != 'native' || matrix.rust-version == '1.33.0'
+      run: "mv Cargo.toml Cargo.toml.bak && sed '/ # benchmark only$/d' Cargo.toml.bak >Cargo.toml"
```

Comment:
After reproducing probably every problem you ran into while writing this line there's only one nit left: Missing space after the `>`.

Reply:
sed is capricious :sweat_smile: